services/ui-src/src/utils/other/parsing.ts

```diff
@@ -51,3 +50,128 @@ export const sanitizeAndParseHtml = (html: string) => {
   const parsedHtml = parse(sanitizedHtml);
   return parsedHtml;
 };
+
+// parsing the field name numbers for the PDF preview page
+export const parseFieldLabel = (labelObject: {
```

Comment:
Post-merge Note: Should refactor to account for fields sometimes having no props, and return a string if so (maybe something like `--`).  Should only return plain text strings without any HTML or styling, which will be handled separately.